### PR TITLE
Fwdport/7.4 beta master

### PR DESCRIPTION
**Pull request by alexanderchan-scality**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -24,19 +29,20 @@
   },
   "homepage": "https://github.com/scality/backbeat#readme",
   "dependencies": {
-    "arsenal": "scality/Arsenal",
+    "arsenal": "scality/Arsenal#fwd/7.4-beta-master",
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

File: package.json [hunk 2]
```diff
@@ -24,19 +29,20 @@
...
     "async": "^2.3.0",
     "aws-sdk": "2.147.0",
     "backo": "^1.1.0",
-    "bucketclient": "scality/bucketclient",
+    "bucketclient": "scality/bucketclient#fwdport/7.4-beta-master",
```
Comment (DavidPineauScality) [inline package.json]: Branch spec needs to be removed for master

**Comment by jonathan-gramain** [comment]
Can you update again with the latest rel/7.4-beta? It will avoid having to do a follow-up PR.

**Comment by rahulreddy** [comment]
@DavidPineauScality Can you please approve the PR if you are satisfied with the changes?

**Pull request merged**